### DISCUSSION TITLE
Dead Project?

### DISCUSSION THREAD

**Discussion by Eutropios** [microsoft/vscode-mypy#44] [General]
Is this project dead? It hasn't received an update in a couple months, and users would greatly appreciate this extension being released.

**Comment by karthiknadig** [comment]
Sorry about the delayed response and work on this. I was on vacation last month. This is still a work in progress project. Currently I am working on switching to `dmypy` see here: https://github.com/microsoft/vscode-mypy/pull/52

**Reply by Eutropios** [reply]
I apologize if my post came off as rude. I do appreciate the work that you put into this project very much. I will definitely give dmypy a look!
